.mk/verify.mk

```diff
@@ -18,13 +18,6 @@ define license_python
 	($(ABSTOOLBIN)/license_finder --python-version=3 || true)
 endef
 
-GO_VERSION:=1.13
```

Comment:
Why is this removed?

Reply:
there are new version of go 1.14 and 1.15, thus no need to limit or bind our project to any specific version (and leave go version and IDE choice to developer).